src/components/ExpensiTextInput/propTypes.js

```diff
@@ -4,6 +4,9 @@ const propTypes = {
     /** Input label */
     label: PropTypes.string,
 
+    /** Text to show if there is an error */
```

Comment:
I think this comment is slightly misleading, since this will be text to show _as_ an error. We do not check `is there an error, then show text` we check `is there error text, then show text`

Reply:
+1

Reply:
A better comment would be `Text to show if there is an error text`?

Reply:
I would just say `Error text to display`

Reply:
Done!